docs/modules/plugins/partials/plugin-web-app-steps.adoc

```diff
@@ -654,3 +655,77 @@ When I find = `5` elements by `By.xpath(//script):a` and for each element do
 |When I set 'type' attribute value of the context element to the 'scenario' variable 'type'|
 |Then `${type}` is equal to `text/javascript`                                              |
 ----
+
+==== Switch to the default context of page
+
+Switches context to the root https://developer.mozilla.org/en-US/docs/Web/HTML/Element/html[<html>] element of the current page.
+
+[source,gherkin]
+----
+When I switch back to the page
+----
+
+.Switch to user context and back to the default context of page
+----
+Given I am on page with URL `https://vividus-test-site.onrender.com/elementState.html`
+When I change context to element located by `id(button-hide)`
+Then text `Element to hide` does not exist
+When I switch back to the page
+Then text `Element to hide` exists
+----
+
+==== Switch context to a frame
+
+Switches to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe[<iframe>] or https://developer.mozilla.org/en-US/docs/Web/HTML/Element/frame[<frame>] element using one of the supported locators.
+
+[source,gherkin]
+----
+When I switch to frame located `$locator`
+----
+* `$locator` - <<_locator>>.
+
+.Switch to frame
+----
+Given I am on page with URL `https://vividus-test-site.onrender.com/nestedFrames.html`
+Then text `Modal Frame Example` does not exist
+When I switch to frame located `id(parent)`
+Then text `Modal Frame Example` exists
+----
+
+==== Switch context to new window
```

Comment:
let's avoid `window` term (it's too technical and not clear for users) and use `tab` instead

Reply:
> let's avoid `window` term (it's too technical and not clear for users) and use `tab` instead

Okay, but since we mention the term `window` in step wording and params, I think it makes sense to give a little explanation of it

Reply:
all "window"-based syntax steps will be updated to use "tab"